openssl.dockerfile

```diff
@@ -34,4 +34,4 @@ RUN <<`
     ln -sf ${OUT_DIR}/lib64/libcrypto.so.3 /lib/x86_64-linux-gnu/libcrypto.so.3
 `
 WORKDIR /
-#RUN rm -rf ${SRC_DIR}
\ No newline at end of file
+RUN rm -rf ${SRC_DIR}
```

Comment:
Did you mean to remove this or uncomment it?

Reply:
Yeah, I was unclear -- just removing the comment, keeping the `RUN`.